docs/tutorials/tut-minemeld-migration.md

```diff
@@ -0,0 +1,291 @@
+Now let's go back to the node's inputs. We need to explore each one. Our example only has one input node listed - let's explore it by clicking the link there `aggregatorIPv4Inbound-clone-MLB`.
+
+![](../doc_imgs/tutorials/tut-minemeld-migration/ipv4-aggregator-node.png)
+
+Let's see the details of the node's prototype by clicking the prototype linked there - `stdlib.aggregatorIPv4Inbound`.
+
+![](../doc_imgs/tutorials/tut-minemeld-migration/ipv4-aggregator-prototype.png)
+
+We see in the `config` section that this prototype filters for indicators whose `type` is 'IPv4' - let's file this information away for when we configure an integration instance in Cortex XSOAR.
+Now if we go back to the aggregator node, we see that it too only has one input node listed. Let's perform the same actions as before and we'll be finished gathering the information we need. Let's click on the input node linked as `exit_addresses-clone-MLB`.
+
+![](../doc_imgs/tutorials/tut-minemeld-migration/exit_addresses-node.png)
+
+Click on the prototype listed there to see more details" `tor.exit_addresses`.
+
+We know from the table mapping MineMeld prototypes to Cortex XSOAR integrations detailed [above](#Minemeld-Prototype-to-Cortex-XSOAR-Integration-Mapping) that the `tor.exit_addresses` prototype maps to the _Tor Exit Addresses Feed_ integration. And with that, we have all the information we need to move forward. Let's review and gather here the information we collected from looking at the output node we want to migrate, and all the nodes that flow into it.
+1. Filters for indicators whose `confidence` is greater than 75 and whose `share_level` is 'green'
+2. Filters for indicators whose `type` is 'IPv4'
+3. The `tor.exit_addresses` prototype maps to the _Tor Exit Addresses Feed_ integration
+
+Let's configure an instance of the __Export Indicators Service__ integration using the information we collected to construct the query that defines which indicators are made available. Search for 'export indicators' in your Cortex XSOAR's integrations page as seen in the following screenshot.
+
+![](../doc_imgs/tutorials/tut-minemeld-migration/search-export-indicators.png)
+
+When configuring an instance of this integration, we need to provide an `Indicator Query`. The value entered here uses the same query syntax one would use in the Cortex XSOAR's indicators page to filter and search for specific indicators.
+
+![](../doc_imgs/tutorials/tut-minemeld-migration/export-indicators-configuration-1.png)
+
+So the information we gathered previously, translated to Cortex XSOAR's indicators query syntax would be,
+```
+type:IP and sourceBrands:"Tor Exit Addresses Feed" and confidence:>75 and trafficlightprotocol:Green
+```
+Enter that value for the **Indicator Query**.
+
+![](../doc_imgs/tutorials/tut-minemeld-migration/export-indicators-configuration-2.png)
+
+Finish configuring the integration to your desired specifications and press `Done`.
+
```

Comment:
Can you add a quick explanation or a screenshot that shows where the list is actually published as a URL or something like that and where the user can see it?

Reply:
it is documented in the integration detailed instructions, dont think we should re-document that